libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -233,7 +234,7 @@ void DocStringTagParser::parseDocStrings(
 
 	for (auto const& [tagName, tagValue]: _annotation.docTags)
 	{
-		string static const customPrefix("custom:");
+		string_view static constexpr customPrefix("custom:");
```

Comment:
C++ 20 seems to allow constexpr strings. http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p0980r1.pdf. Maybe something to change once we upgrade to C++20.